backend/src/main/java/pt/ulisboa/tecnico/socialsoftware/tutor/impexp/domain/CSVQuizExportVisitor.java

```diff
@@ -27,9 +29,12 @@ public String export(Quiz quiz, List<QuestionAnswerItem> questionAnswerItems) {
         int lineSize = numberOfQuestions + 5;
         Map<Integer, QuizQuestion> quizQuestions = quiz.getQuizQuestions().stream()
                 .collect(Collectors.toMap(QuizQuestion::getId, Function.identity()));
+        // TODO[is->has]: questionMap XML
         Map<Integer, Option> options = quiz.getQuizQuestions().stream()
                 .map(QuizQuestion::getQuestion)
-                .flatMap(question -> question.getOptions().stream())
+                .map(Question::getQuestionDetails)
```

Comment:
But we need to have like a standard format no? Is this being used for anything now?
My suggestion would be to for instance have a "answer representation":
- multiple choice would be A;B;C or D. 
- In code questions it could be the combination like (A,B,B,D) (Dropdown 1 = Option A (first)) and so on...
Not I am not sure if this would work

Reply:
We could do this with the visitor then...